GNUmakefile

```diff
@@ -58,7 +58,7 @@ cov:
 
 test: format
 	@echo "--> Running go test" ;
-	@go test $(PACKAGES)
+	@go test -cover $(PACKAGES)
```

Comment:
This might increase the overall time. I feel it better to have a separate make target. & have good comments on what that target does.